disassemblers/ofrak_ghidra/ofrak_ghidra/ghidra_model.py

```diff
@@ -4,6 +4,7 @@
 
 import aiohttp
 
+from ofrak import ResourceFilter
```

Comment:
Delete this line or 10, one of these is an unneeded/unused import - did this pass `autoflake`?